pages/pageObjectProject/productPage.js

```diff
@@ -0,0 +1,23 @@
+export function createProductPage(page) {
+  const productItem = page.locator('.product-item-info');
+  const size = page.locator('[option-label="M"]');
+  const color = page.locator('[option-label="Blue"]');
+  const buttonAddToCart = page.locator('button[title="Add to Cart"]');
+  const successMessage = page.locator('[data-ui-id="message-success"]');
+
+  async function addToCart() {
```

Comment:
вот тут `size` и `color` лучше сделать параметрами функции, можно со значениями по-умолчанию, если часто один и тот же добавляется

```suggestion
  async function addToCart({ size = 'M', label = 'Blue' } = {}) {
```